content/src/content/jcr_root/apps/core/wcm/components/image/v3/image/README.md

```diff
@@ -47,20 +47,23 @@ If any other value than the default one is configured, that value will be used t
 The following properties are written to JCR for this Image component and are expected to be available as `Resource` properties:
 
 1. `./fileReference` property or `file` child node - will store either a reference to the image file, or the image file
-2. `./isDecorative` - if set to `true`, then the image will be ignored by assistive technology
-3. `./alt` - defines the value of the HTML `alt` attribute (not needed if `./isDecorative` is set to `true`)
-4. `./linkURL` - allows defining a URL to which the image will link to
-5. `./width` - allows defining a HTML `width` attribute, useful for browser to calculate the aspect ratio of the image, preventing the layout shifts
-6  `./height` - allows defining a HTML `height` attribute, useful for browser to calculate the aspect ratio of the image, preventing the layout shifts
-7. `./jcr:title` - defines the value of the HTML `title` attribute or the value of the caption, depending on the value of
+1. `./isDecorative` - if set to `true`, then the image will be ignored by assistive technology
+1. `./alt` - defines the value of the HTML `alt` attribute (not needed if `./isDecorative` is set to `true`)
+1. `./altValueFromDAM` - if `true`, the HTML `alt` attribute is inherited from the DAM asset.
+1. `./altValueFromPageImage` - if `true`, the HTML `alt` attribute is inherited from the featured image of the page. This property is only enabled when `./imageFromPageImage` is `true`.
+1. `./imageFromPageImage` - if `true`, the image is inherited from the featured image of the page.
```

Comment:
I would rather we call this `imageInherited`

Reply:
I'd prefer to keep the current name as it is in line with the conventions that have been used before, e.g. `alt` and `altValueFromDAM`.